plugins/processors/gpu/processor.go

```diff
@@ -0,0 +1,194 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT
+
+package gpu
+
+import (
+	"context"
+	"encoding/json"
+	"strings"
+
+	"go.opentelemetry.io/collector/component"
+	"go.opentelemetry.io/collector/pdata/pcommon"
+	"go.opentelemetry.io/collector/pdata/pmetric"
+	"go.uber.org/zap"
+)
+
+const (
+	gpuMetric                = "_gpu_"
+	gpuContainerMetricPrefix = "container_"
+	gpuPodMetricPrefix       = "pod_"
+	gpuNodeMetricPrefix      = "node_"
+)
+
+var podContainerMetricLabels = map[string]map[string]interface{}{
+	"ClusterName":  nil,
+	"FullPodName":  nil,
+	"PodName":      nil,
+	"InstanceId":   nil,
+	"InstanceType": nil,
+	"NodeName":     nil,
+	"Timestamp":    nil,
+	"Type":         nil,
+	"Version":      nil,
+	"Namespace":    nil,
+	"Sources":      nil,
+	"UUID":         nil,
+	"kubernetes":   nil,
```

Comment:
nit: Is there a reason this attribute is differently cased than the others?

Reply:
I don't have the history of these dimension names, but CI already uses these so I wouldn't change it.